Sources/TuistKit/Models/Scheme.swift

```diff
@@ -1,7 +1,7 @@
 import Basic
 import Foundation
 
-class Scheme: JSONMappable, Equatable {
```

Comment:
It doesn't seem the `Scheme` is used anywhere at the moment - was the idea to associate with `Project` at a later time to help customize scheme generation?

Reply:
I think so. If we are not using it at the moment I'd get rid of it entirely.

Reply:
Ah - while not used yet in the code base, there's a pending PR https://github.com/tuist/tuist/pull/200 to make use of it!

Reply:
will push an update to include the json conversion for it to help ease resolving conflicts on that PR